docs/rst/manual/about/release_notes.rst

```diff
@@ -1,6 +1,70 @@
 Release Notes
 =============
 
+Version 2.14
+------------
+
+Highlighted changes
+~~~~~~~~~~~~~~~~~~~
+
+Restarting ES-MDA
+#################
+
+It is now possible to restart ES-MDA runs from an intermediate iteration. Note
+that this requires a bit of care due to the lack of metadata in current storage.
+We are aiming at resolving this in the future in the new storage that soon will
+be the standard.
+
+After selecting the ES-MDA algorithm, you first need to set `Current case` to
+the case you intend to be your first case to reevaluate. After which the
+iteration number will be wrongly injected into the `Target case format`, which
+you have to remove manually (reset it to the original `Target case format`).
+After which you have to set `Start iteration` to the iteration for which you
+are to start from; this number must correspond to the iteration number of the
+case you have selected as your `Current case`. We recognize that this induces
+some manual checking, but this is due to the lack of metadata mentioned above.
+We still hope that this can aid our users and prevent the need of a complete
+restart due to cluster issues etc.
```

Comment:
This is excellent. Is it also in the docs? I can't find it.

Reply:
Unfortunately not @jondequinor. I can't find a good place to put it to be honest, as we have very little documentation of the GUI to begin with. We could write a section on reevaluating and restarting. Similar functionality was added for the ensemble smoother earlier this year and maybe it would fit nicely together?